src/osd/osd_types.h

```diff
@@ -796,9 +796,7 @@ class eversion_t {
   explicit eversion_t(bufferlist& bl) : __pad(0) { decode(bl); }
 
   static eversion_t max() {
-    eversion_t max;
-    max.version -= 1;
-    max.epoch -= 1;
+    static eversion_t max(-1,-1);
```

Comment:
make this variable a `const`. and change the signature of this method like
```c++
static const eversion_t& max() 
```

but since this is not a pure win. do you observe the performance boost with this change? if yes, could you share any statistics with us?